docs_src/getting-started/quick-start/index.md

```diff
@@ -33,180 +37,180 @@ docker-compose ps
 
 ``` yaml
   app-service-mqtt:
-    image: edgexfoundry/docker-app-service-configurable:1.1.0
-    ports:
-      - "127.0.0.1:48101:48101"
-    container_name: edgex-app-service-configurable-mqtt
-    hostname: edgex-app-service-configurable-mqtt
-    networks:
-      - edgex-network
-    environment:
-      <<: *common-variables
-      edgex_profile: mqtt-export
-      Service_Host: edgex-app-service-configurable-mqtt
-      Service_Port: 48101
-      MessageBus_SubscribeHost_Host: edgex-core-data
-      Binding_PublishTopic: events
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Address: broker.mqttdashboard.com
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Port: 1883
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Protocol: tcp
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Publisher: edgex
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Topic: EdgeXEvents
+    container_name: edgex-app-mqtt
     depends_on:
-      - consul
-      - data
+    - consul
+    - data
+    environment:
+      CLIENTS_CORE_COMMAND_HOST: edgex-core-command
+      CLIENTS_CORE_DATA_HOST: edgex-core-data
+      CLIENTS_CORE_METADATA_HOST: edgex-core-metadata
+      CLIENTS_SUPPORT_NOTIFICATIONS_HOST: edgex-support-notifications
+      CLIENTS_SUPPORT_SCHEDULER_HOST: edgex-support-scheduler
+      DATABASES_PRIMARY_HOST: edgex-redis
+      EDGEX_PROFILE: mqtt-export
+      EDGEX_SECURITY_SECRET_STORE: "false"
+      MESSAGEQUEUE_HOST: edgex-redis
+      REGISTRY_HOST: edgex-core-consul
+      SERVICE_HOST: edgex-app-mqtt
+      TRIGGER_EDGEXMESSAGEBUS_PUBLISHHOST_HOST: edgex-redis
+      TRIGGER_EDGEXMESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-redis
+      WRITABLE_PIPELINE_FUNCTIONS_MQTTEXPORT_PARAMETERS_BROKERADDRESS: tcp://broker.mqttdashboard.com:1883
+      WRITABLE_PIPELINE_FUNCTIONS_MQTTEXPORT_PARAMETERS_TOPIC: EdgeXEvents
+    hostname: edgex-app-mqtt
+    image: edgexfoundry/app-service-configurable:2.0.0
+    networks:
+      edgex-network: {}
+    ports:
+    - 127.0.0.1:59702:59702/tcp
+    read_only: true
+    security_opt:
+    - no-new-privileges:true
+    user: 2002:2001
 ```
 
 !!! Note
-    This adds the application service configurable to your EdgeX system.  The application service configurable allows you to configure (versus program) new exports - in this case exporting the EdgeX sensor data to the HiveMQ broker at broker.mqttdashboard.com port 1883.  You will be publishing to EdgeXEvents topic.
+    This adds the application service configurable to your EdgeX system.  The application service configurable allows you to configure (versus program) new exports - in this case exporting the EdgeX sensor data to the HiveMQ broker at `tcp://broker.mqttdashboard.com:1883`.  You will be publishing to the EdgeXEvents topic.
```

Comment:
Do we also want a note about building custom compose files with link to this section?: https://docs.edgexfoundry.org/2.0/getting-started/Ch-GettingStartedUsers/#generate-a-custom-docker-compose-file